cms/models.py

```diff
@@ -446,6 +455,22 @@ class InstructorsSection(BootcampRunChildPage):
     ]
 
 
+class InstructorsSection(InstructorSponsorSection):
+    """
+    InstructorsPage representing a "Your MIT Instructors" section on a product page
+    """
+
```

Comment:
Just a suggestion: Moving the sections field defined in InstructorSponsorSection to InstructorsSection would make it consistent and cleaner with SponsorsSection.

Reply:
I initially decided to keep it there as part of the base class since I was going to do this in a different way. But i think what you suggested would increase the readability and there would be no change in the migration or the data itself. I'll do this change and let you know.

Reply:
Done in https://github.com/mitodl/bootcamp-ecommerce/pull/1454/commits/ff6cb382e9dc2d506e1f45db21111c1fda656842. Also moved sections `content_panels` a little down in the hierarchy because that would make sense with this change.